crates/bevy_ui/src/entity.rs

```diff
@@ -96,7 +96,7 @@ impl Default for TextBundle {
 }
 
 /// A UI node that is a button
-#[derive(Bundle, Clone, Debug)]
+#[derive(Bundle, Default, Clone, Debug)]
```

Comment:
We should consider splitting this change out into its own PR and merging it ASAP.

Reply:
Okay, I'll remove it from this one